soccer/src/soccer/soccermom.cpp

```diff
@@ -0,0 +1,30 @@
+#include "soccermom.hpp"
+
+#include <spdlog/spdlog.h>
+
+namespace tutorial {
+
+SoccerMom :: SoccerMom() : Node("soccer_mom") {
+       publisher_ = this->create_publisher<std_msgs::msg::String>("team_fruit", 10);
+       subscription_ = this->create_subscription<rj_msgs::msg::TeamColor>(
+      "referee/team_color", 10, [this](rj_msgs::msg::TeamColor::SharedPtr color) {
+          is_blue_ = color->is_blue;
+          });
+       timer_ = create_wall_timer(std::chrono::milliseconds(16), [this]() { send(); });
```

Comment:
interesting implementation. a timer is fine, but in this case it is best to only publish in reaction to the subscription, as opposed to every 16 ms. you only need to publish when the data has changed.